src/components/Table/TableFactory.js

```diff
@@ -34,7 +34,7 @@ export const TableFactory = props => {
 
   const onTableChange = (rows, data) =>
     onChange(rows, data, props.onRowUpdate, props.onRowDeleteOrMove, props.onRowActivate);
-  const error = props.error !== '' ? <Alert>{props.error}</Alert> : undefined;
+  const error = props.error !== undefined ? <Alert>{props.error}</Alert> : undefined;
```

Comment:
Shouldn't it be `props.error ? <Alert/> : null` ?

Or even better - change L43:
```
{props.error && <Alert> ...}
```

Reply:
@mareklibra fixed